plugins/modules/azure_rm_eventhub.py

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Praveen Ghuge(@praveenghuge) Karl Dasan(@karldas30) Saurabh Malpani (@saurabh3796)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: azure_rm_eventhub
+version_added: "1.6.0"
+short_description: Manage Event Hub
+description:
+    - Create, update and delete instance of Event Hub.
+options:
+    resource_group:
+        description:
+            - Name of the resource group to which the resource belongs.
+        required: True
+        type: str
+    namespace_name:
+        description:
+            - Name of the namespace in which to create event hub.
+        required: True
+        type: str
+    name:
+        description:
+            - Unique name of the Event Hub.
+        required: False
+        type: str
+    message_retention_in_days:
+        description:
+            - Number of days to retain the events for this Event Hub.
+        required: False
+        type: int
+    partition_count:
+        description:
+            - Number of partitions created for the Event Hub.
+        required: False
+        type: int
+        choices: Choose value between 1 to 32(both included).
```

Comment:
According to Ansible Sanity definition specification should be a list of 1~32! Thank you very much!
```suggestion
        choices:  [1,2,3,4,5,6,7,8,......,31,32]
```